webserver/webserver/src/main/java/io/helidon/webserver/RequestRouting.java

```diff
@@ -380,6 +380,10 @@ private void defaultHandler(Throwable t) {
 
                     response.status(Http.Status.INTERNAL_SERVER_ERROR_500);
                 }
+                String message = t.getMessage();
+                if (message != null) {
+                    response.send(message);
```

Comment:
How does that play with the response.send() below ?

Reply:
Forgot to add WIP to the title. But that needs to be fixed.

Reply:
Oh ok, no problem.